src/content/api/compilation-hooks.md

```diff
@@ -464,6 +465,50 @@ The assets have been optimized.
 - Callback Parameters: `assets`
 
 
+### `processAssets`
+
+`AsyncSeriesHook`
```

Comment:
types defined in https://github.com/webpack/webpack/blob/a4d06805f2f934132ee14229a73aaa5c532a822b/types.d.ts#L979